app/components/HeaderBare.vue

```diff
@@ -0,0 +1,31 @@
+<template>
```

Comment:
Nice to have a Bare bones template. Good idea!

Maybe we can use this to generate a non-js page for those hackers out there blocking scripts on the web in the future?

Reply:
it is just the header that is shown on error page, without the menu etc.